app/forms/recovery_code_form.rb

```diff
@@ -25,9 +25,9 @@ def valid_recovery_code?
     recovery_code_generator.verify(code)
   end
 
-  def result
+  def extra_analytics_attributes
     {
-      success: success,
+      method: 'recovery code',
```

Comment:
With a constant, we would end up with `method: RECOVERY_CODE_MFA_METHOD`, which does not make things clearer IMO, and perhaps makes them less clear.